test_scripts/API/commonSetAppIcon.lua

```diff
@@ -0,0 +1,110 @@
+---------------------------------------------------------------------------------------------------
```

Comment:
@HSavynetska Why do we need 3 common modules? Or these are duplicates and need to be removed?

Reply:
Duplicates